docs/tryout.md

```diff
@@ -0,0 +1,390 @@
+# User Managed Access with WSO2 Identity Server
+
+WSO2 Identity Server (WSO2 IS) supports the UMA 2.0 protocol, which allows a resource owner to easily share resources with other requesting parties. This tutorial demonstrates how you can use WSO2 IS as the authorization server to try out UMA for a sample scenario.
+
+## Introduction
+
+To demonstrate the UMA flow using WSO2 IS as the authorization server, let’s consider a sample scenario where the resource owner wants to share a resource on their resource server with a requesting party. This requesting party uses a client to view the resource owner's resource. Note that the resource owner wants to allow this requesting party to only have view permission to their resource.
+
+Once you try out the guide you will understand how to use UMA 2.0 to do the following:
+
+-   Outsource authorization.
+-   Control resources and share resources in a selective manner via a
+    single console (authorization server).
+
+---
+
+## Create the resource owner
+
+
+1.  Log in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) using administrator credentials (`admin:admin`).
+2.  On the **Main** > **Identity** section, click **Add** under **Users and Roles**.
+<a name ="resourceowner_credentials"></a>
```

Comment:
is this a link?

Reply:
No its not

Reply:
oh.. it renders as `admin:admin`. Nice.
Not sure how you got that to work.. :)